lib/data/process-test-directory/index.js

```diff
@@ -86,7 +87,10 @@ function validateTestsKeys(row) {
     throw new Error('testId does not match the expected format: ' + row.testId);
   if (!titleFormat.test(row.title))
     throw new Error('title does not match the expected format: ' + row.title);
-  if (row.assertions && !assertionsFormat.test(row.assertions))
+  if (
+    row.assertions &&
+    (!assertionsFormat.test(row.assertions) || invalidPriorityFormat.test(row.assertions))
+  )
```

Comment:
@howard-e I was able to bring back the priority validation by simply running two regex tests :-)

Reply:
Nice!